src/test/scala/edu/ie3/osmogrid/cfg/OsmoGridConfigFactory.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.osmogrid.cfg
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+import java.io.File
+import scala.util.Try
+
+object OsmoGridConfigFactory {
```

Comment:
Why? 
I mean I see what it does but I don't see the reason why we need this...?

Reply:
Just for testing. And to reduce code duplications in test classes.